src/components/Dashboard/Dashboard.jsx

```diff
@@ -15,46 +16,21 @@ export default function Dashboard() {
 
   const handleAdd = (diary) => {
     const newDiary = { ...diary, id: uuidv4() };
-    setDiaries([newDiary, ...diaries]);
+    addDiary(newDiary);
     toggleNewDiaryForm();
   };
 
-  const handleDelete = (id) => {
-    setDiaries(diaries.filter((diary) => diary.id !== id));
-  };
-
-  const onSave = (updatedDiary) => {
-    setDiaries((currentDiaries) =>
-      currentDiaries.map((diary) =>
-        diary.id === updatedDiary.id ? updatedDiary : diary
-      )
-    );
-  };
-
-  // const storedDiaries = localStorage.getItem("diaries");
-  // if (storedDiaries) { // truthy : [] <-> null undefined
-  //   setDiaries(JSON.parse(storedDiaries));
-  // } -> (한번만) 저장되어있는 것을 GET! 상태를 업데이트하고 useEffect...
-  // 내가 쓸 로직이 함수 안에 혹은 밖에 있어야 하는가 확인!
-
-  useEffect(() => {
-    localStorage.setItem("diaries", JSON.stringify(diaries));
-  }, [diaries]);
+  useEffect(() => {}, [diaries]);
 
   return (
     <div className={styles.diaryList}>
       {showNewDiaryForm && (
         <NewDiaryForm onAdd={handleAdd} className={styles.newDiaryForm} />
       )}
-      <DiaryList diaries={diaries} onDelete={handleDelete} onSave={onSave} />
+      <DiaryList diaries={diaries} onDelete={deleteDiary} />
```

Comment:
no need to pass `deleteDiary`

Reply:
deleted onDelete and deleteDiary

Reply:
resolved https://github.com/Gracekangdh/DiaryApp/pull/1/commits/75209445563bf42338df5bbeae5ee06b9f6ffc35